packages/dropzone/README.md

```diff
@@ -104,6 +104,22 @@ When a file is dragged over the `<sp-dropzone>` element, it will display with th
 </sp-dropzone>
 ```
 
+### Filled
+
+When the `<sp-dropzone>` is in a filled state, set the `filled` attribute, as follows:
+
+```html
+<style>
+    sp-action-button {
+        margin-block-end: 32px;
+    }
+</style>
+<sp-action-button draggable="true">Drag me!</sp-action-button>
+<sp-dropzone id="dropzone" filled>
+    <div>Drop file here to replace</div>
```

Comment:
Does this text change to surface up the notification that 'your file has been dropped'?